_clients/logstash/index.md

```diff
@@ -57,6 +57,9 @@ The OpenSearch Logstash plugin has two installation options at this time: Linux
 
 Make sure you have [Java Development Kit (JDK)](https://www.oracle.com/java/technologies/javase-downloads.html) version 8 or 11 installed.
 
+If you're migrating from from Elasticsearch or Open Distro for Elasticsearch, make sure you use the `opensearch` Logstash output plugin.
```

Comment:
I think we want to say something like "If you're migrating from an existing Logstash installation, you can install the [OpenSearch output plugin](https://rubygems.org/gems/logstash-output-opensearch/) manually and [update pipeline.conf](https://opensearch.org/docs/latest/clients/logstash/ship-to-opensearch/). We include this plugin by default in our tarball and Docker downloads."

The "make sure you use the output plugin" doesn't really clarify if people need to download and install something or just change their pipeline to use the plugin. Let me know what you think.